package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "electron": "1.4.1",
     "electron-edge": "6.5.4",
-    "gpii-windows": "git://github.com/danailbd/windows.git#hst-2017",
+    "gpii-windows": "git://github.com/danailbd/windows.git",
```

Comment:
Is there something special in this branch which is not yet in trunk? We should try to get merged up

Reply:
The problem is that the master branch of GPII-windows repo is [pointing to some commit](https://github.com/GPII/windows/blob/master/package.json#L15) of `universal` that is missing the `pcpChannel`, instead of master. In our fork, we've change it to use master. Probably we'd want to apply this change in the main windows repo as well?

Reply:
This got updated recently, can we recheck it?